config/initializers/audit.rb

```diff
@@ -5,3 +5,10 @@
 
 Audit = Audited.audit_class
 Audit.send(:include, AuditExtensions)
+
+# if this is a migration rake task we disable auditing in particular models
+if Foreman.in_rake?('db:migrate')
+  [ Host::Managed, Hostgroup, ProvisioningTemplate, Ptable, User ].each do |klass|
```

Comment:
@domcleal audited method would have to exist in AR:Base prior aliasing. If I defined some placeholder (so I could do the aliasing) then Audited wouldn't work again, because it's also a module and it does not override existing `audited` method (it would have to be implemented as alias). Or am I missing something?